website/versioned_docs/version-v1.x/circuits.md

```diff
@@ -427,24 +427,30 @@ Before building the project, make sure you have the following dependencies insta
 To build the two main circuits of MACI, run the following commands:
 
 ```
-circom --r1cs --sym --wasm --output ./build circom/test/processMessages_test.circom
-circom --r1cs --sym --wasm --output ./build circom/test/tallyVotes_test.circom
+circom --r1cs --sym --wasm --output ./build circom/test/processMessages_10-2-1-2_test.circom
+circom --r1cs --sym --wasm --output ./build circom/test/tallyVotes_10-1-2_test.circom
 ```
 
 Please note that the circuit is configured with testing purpose parameters, which means it can only handle a limited amount of messages (up to 25 messages). For more information on the parameters and how to configure them, refer to [this page](/docs/circuits/#compile-circuits).
 
 ### Generating zKeys
 
-Run:
+Run from the root directory to save to the `cli/zkeys` folder:
 
 ```bash
-pnpm exec zkey-manager compile -c ./zkeys.config.yml
+pnpm setup:zkeys
+```
+
+Run from the circuits folder with `--outPath` to save to a custom folder:
+
+```bash
+cd circuits && pnpm gen-zkeys --outPath ./customFolder
```

Comment:
I think the name doesn't matter, should it be something like `./CUSTOM_FOLDER_NAME` to let the user know they could setup whatever name for that directory.

Reply:
definitely, that might be easier to understand